cassandra/io/asyncorereactor.py

```diff
@@ -102,7 +102,7 @@ def handle_read(self):
     def notify_loop(self):
         if not self._notified:
             self._notified = True
-            os.write(self.write_fd, 'x')
+            os.write(self.write_fd, b'x')
```

Comment:
You are correct. I noticed the issue as I was testing these changes across all versions.